src/sw/helpers/ServiceWorkerUtilHelper.ts

```diff
@@ -33,4 +33,29 @@ export default class ServiceWorkerUtilHelper {
     }
     return availableWorker;
   }
+
+  // Allows waiting for the service worker registration to become active.
+  // Some APIs, like registration.pushManager.subscribe, required it be active
+  // otherwise it throws.
+  static waitForServiceWorkerToActive(
```

Comment:
Optional nit: `waitForActiveServiceWorker` or `waitForActiveSW`

Reply:
ah ya, name is confusing, i think `waitUntilActive` is a better name.
* This function is a static function so the caller code will be `ServiceWorkerUtilHelper.waitUntilActive(registration)`